src/Aspire.Hosting.AWS/README.md

```diff
@@ -157,6 +157,107 @@ builder.AddProject<Projects.Frontend>("Frontend")
        .WithReference(construct, c => c.Url, "Url");
 ```
 
+## Integrating AWS Lambda Local Development
+
+You can develop and test AWS Lambda functions locally within your .NET Aspire application. This enables testing Lambda functions alongside other application resources during development.
+
+![\[\]!(.)](Resources/lambda.png)
+
+
+### Adding Lambda Functions
+
+To add a Lambda function to your .NET Aspire AppHost, use the `AddAWSLambdaFunction` method. The method supports both executable Lambda functions and class library Lambda functions:
+
+```csharp
+#pragma warning disable CA2252 // This API requires opting into preview features
+
+// Add an executable Lambda function
+builder.AddAWSLambdaFunction<Projects.ExecutableLambdaFunction>(
+    "MyLambdaFunction", 
+    handler: "ExecutableLambdaFunction")
+    .WithReference(awsConfig);
+
+// Add a class library Lambda function
+builder.AddAWSLambdaFunction<Projects.ClassLibraryLambdaFunction>(
+    "MyLambdaFunction", 
+    handler: "ClassLibraryLambdaFunction::ClassLibraryLambdaFunction.Function::FunctionHandler")
+    .WithReference(awsConfig);
+
+```
+
+The handler parameter specifies the Lambda handler in different formats depending on the project type:
+
+- For executable projects: specify the assembly name.
+- For class library projects: use the format `{assembly}::{type}::{method}`.
+
+### API Gateway Local Emulation
+
+To add an API Gateaway emulator to your .NET Aspire AppHost, use the `AddAPIGatewayEmulator` method. 
```

Comment:
Can you add a blurb about using the wild card paths with `{proxy+}`. I suspect that will be a common scenario especially with the ASP.NET Core bridge library. In fact you might want to give an example how to do the API Gateway setup for the ASP.NET Core bridge. Below is an example where you need to add the root and wildcard.

```
var aspNetCoreFUnction = builder.AddAWSLambdaFunction<Projects.AWSServerless51>("Resource", "AWSServerless51");

builder.AddAWSAPIGatewayEmulator("api", Aspire.Hosting.AWS.Lambda.APIGatewayType.Rest)
        .WithReference(aspNetCoreFUnction, Aspire.Hosting.AWS.Lambda.Method.Any, "/")
        .WithReference(aspNetCoreFUnction, Aspire.Hosting.AWS.Lambda.Method.Any, "/{proxy+}");
```

Reply:
updated